docs/core_docs/src/theme/Feedback.js

```diff
@@ -110,7 +111,9 @@ const getIpAddress = async () => {
 
 export default function Feedback() {
   const { setCookie, checkCookie } = useCookie();
+  const [feedbackId, setFeedbackId] = useState(null);
```

Comment:
Might make more sense to store the ID in the cookie which is tracking feedback, this way they can't spam it, and their ID will persist between refreshes (for 24hrs)

Reply:
They can't visually get back into that state anyway